lib/codelog/output/log.rb

```diff
@@ -0,0 +1,13 @@
+module Codelog
+  module Output
+    class Log
```

Comment:
To make the interface of this class the same as the other output (and use dependency injection), i could add a second parameter, but it would violate the   Interface Segregation Princible, because i would be adding a interface i won't use, just to make these classes's signature equal